From c4bf65fd069502e699f5ec3473dd99e2fb6848ba Mon Sep 17 00:00:00 2001
From: Do Quang Thang <dq-thang@jinso.co.jp>
Date: Fri, 15 Feb 2013 13:37:03 +0900
Subject: [PATCH 709/859] ASoC: shmobile: armadillo800eva: enable clock
 inversion on FSI-A

Since board flags were updated with simple-card driver in upstream kernel on
a4a2992c531f6ca0aa00ce0deb31e51c1b7ae69b
(ASoC: simple-card: add asoc_simple_dai for initializing)
fixup patch for an "old style" does not exist.
This patch is for LTSI kernel

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Tested-by: Nguyen Viet Dung <nv-dung@jinso.co.jp>
---
 arch/arm/mach-shmobile/board-armadillo800eva.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-shmobile/board-armadillo800eva.c b/arch/arm/mach-shmobile/board-armadillo800eva.c
index 72690cd..32c5c53 100644
--- a/arch/arm/mach-shmobile/board-armadillo800eva.c
+++ b/arch/arm/mach-shmobile/board-armadillo800eva.c
@@ -795,6 +795,7 @@ static int fsi_hdmi_set_rate(struct device *dev, int rate, int enable)
 static struct sh_fsi_platform_info fsi_info = {
 	/* FSI-WM8978 */
 	.port_a = {
+		.flags = SH_FSI_BRS_INV,
 		.tx_id = SHDMA_SLAVE_FSIA_TX,
 	},
 	/* FSI-HDMI */
-- 
1.8.1.4

